assets/js/modules/optimize/index.js

```diff
@@ -32,6 +33,23 @@ import { MODULES_OPTIMIZE } from './datastore/constants';
 export { registerStore } from './datastore';
 
 export const registerModule = ( modules ) => {
+	/**
+	 * Add data to the congrats setup Win Notification for display.
+	 */
+	addFilter(
+		'googlesitekit.SetupWinNotification-optimize',
+		'googlesitekit.OptimizeSetupWinNotification',
+		( winData ) => {
+			winData.description = __(
+				'To set up experiments and see the results, go to ',
+				'google-site-kit'
+			);
+			winData.learnMore.label = 'Optimize';
+			winData.learnMore.url = 'https://optimize.withgoogle.com/';
+			return winData;
+		}
+	);
```

Comment:
This looks a bit out of place here but I like the idea of keeping the module index side-effect free.

Reply:
Yeah, it felt weird but like the least-worst-option. I'll do this for all modules, but I'll annotate it as legacy 👍🏻